src/subcommand/wallet/list.rs

```diff
@@ -0,0 +1,92 @@
+use {super::*, bitcoincore_rpc::RpcApi};
+
+pub(crate) fn run(options: Options) -> Result {
+  let index = Index::open(&options)?;
+  index.index()?;
+
+  let client = options.rpc_client()?;
+
+  let unspent = client.list_unspent(None, None, None, None, None)?;
+
+  let mut utxos = Vec::new();
+  for utxo in unspent {
+    let output = OutPoint::new(utxo.txid, utxo.vout);
+    match index.list(output)? {
```

Comment:
I think we should implement a method, `Index::list_unspent` which is like `Index::List`, but which returns `Result<Vec<(u64, u64)>>`, i.e, not a `List` enum. It should call `Index::list` internally, and return an error if the output is spent or not seen. Then, we can use this in `ord wallet list` and `ord wallet identify`, and de-duplicate the error handling code.